.docker/php/getlaminas.ini

```diff
@@ -25,3 +25,4 @@ opcache.enable_cli=0
 opcache.save_comments=1
 opcache.load_comments=1
 opcache.error_log=
+opcache.preload = /var/www/preload.php
```

Comment:
Considering the docker configuration _is for development_, I don't think we want this.